runtime/runtime.go

```diff
@@ -4,20 +4,23 @@
 
 package runtime
 
+// Contains parts of the runtime package common to all platforms
```

Comment:
This comment can be removed when the shared object loading is factored into a separate internal package.